automation/project/Settings.scala

```diff
@@ -27,7 +27,7 @@ object Settings {
   )
 
   val testSettings = List(
-    testOptions in Test += Tests.Argument("-oF", "-u", "test-reports")
+    testOptions in Test += Tests.Argument("-oFDG", "-u", "test-reports", "-fW", "test-reports/summary.log")
```

Comment:
`D` - Show test duration after test name.
`G` - Show all failed tests and their full stacktrace at the end of log file so I don't have to scroll to find stacktrace.
`-fW` - Write logs in summary.log without colored text.